downstairs/src/lib.rs

```diff
@@ -1645,6 +1619,22 @@ where
         .await
 }
 
+async fn reply_task<WT>(
+    mut resp_channel_rx: mpsc::Receiver<Message>,
+    mut fw: FramedWrite<WT, CrucibleEncoder>,
+) -> Result<()>
+where
+    WT: tokio::io::AsyncWrite
+        + std::marker::Unpin
+        + std::marker::Send
+        + 'static,
+{
+    while let Some(m) = resp_channel_rx.recv().await {
+        fw.send(m).await?;
```

Comment:
Note to self, this would be a great place for a counter of jobs responded to by a downstairs.